README.md

```diff
@@ -19,26 +20,48 @@ Please check out our [Contributor's guide](./CONTRIBUTORS.md) on how you can act
 
 # Credits and Special Thanks
 
-Full credits can be found in-game, or wherever the credits.json file is.
+The full credits can be found in-game, or wherever the credits.json file is.
 
 ## Programming
 - [ninjamuffin99](https://twitter.com/ninja_muffin99) - Lead Programmer
 - [EliteMasterEric](https://twitter.com/EliteMasterEric) - Programmer
 - [MtH](https://twitter.com/emmnyaa) - Charting and Additional Programming
+- [Jenny Crowe](https://twitter.com/JennyCrowe) - Additional Programming
+- [hazel](https://twitter.com/hazel) - Additional Programming, Devops and Additional Internal Tooling
+- [Mike Welsh](https://twitter.com/MikeWelsh) - Additional Programming
+- [Saharan](https://twitter.com/Saharan) - Additional Programming
+- [Ian Harrigan](https://twitter.com/IanHarrigan) - Additional Programming
+- [Osaka Red LLC: Thomas J Webb](https://twitter.com/ThomasJWebb) - Additional Programming

```

Comment:
I believe that the aforementioned people here have been attached to incorrect Twitter accounts or accounts that simply don't exist. 

Maybe someone from the team should chime in and examine and see if they can correct the mistakes and say who does or doesn't have Twitter and/or any other social media platform.